Tests/AppTests/Controllers/User/UserControllerTests.swift

```diff
@@ -0,0 +1,180 @@
+import XCTest
+import Testing
+import HTTP
+import Sockets
+@testable import Vapor
+@testable import App
+
+class UserControllerTests: TestCase {
+  let drop = try! Droplet.testable()
+  
+  let firstName = "Cocoa"
+  let lastName = "Heads"
+  let company = "CocoaHeads"
+  let position = "Tester"
+  let email = "tester@cocoaheads.ru"
+  let phone = "1234567890"
+  let photo = "http://cocoaheads.ru/photo.jpg"
+  let updatedLastName = "Feet"
+  
```

Comment:
You may use  String+Random methods here like String.randomValue, String.randomEmail, randomPhone etc